content/articles/2019-08-callr-task-q.Rmd

```diff
@@ -0,0 +1,586 @@
+---
+title: Multi Process Task Queue in 100 Lines of R Code
+slug: callr-task-q
+author: Gábor Csárdi
+date: '2019-08-03'
+description: >
+  We use `callr::r_session` to implement a worker pool and task queue in
+  100 lines of R code.
+categories:
+- programming
```

Comment:
Are these existing categories? I think I'd prefer something like "deep-dive" for this sort of post.

Reply:
"programming" is, I'll use "deep-dive" instead of "parallelism". It will be a new category, I'll try to find some fitting icon for it.